src/main/java/com/entropyteam/entropay/employees/common/BaseService.java

```diff
@@ -0,0 +1,55 @@
+package com.entropyteam.entropay.employees.common;
```

Comment:
I'd move this interface to something like com.entropyteam.entropay.common.services, in order to keep it outside employees or any other module-specific package.